miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/RetrofitCreatorUtils.kt

```diff
@@ -64,19 +64,21 @@ private fun provideHeaderInterceptor(): Interceptor = Interceptor { chain ->
     chain.proceed(request)
 }
 
-private fun createCertificatePinner(baseUrl: String, pubKeyList: List<String>): CertificatePinner {
+@VisibleForTesting
+internal fun createCertificatePinner(baseUrl: String, pubKeyList: List<String>): CertificatePinner {
     val certificatePinnerBuilder = CertificatePinner.Builder()
     for (pubKey in pubKeyList) {
         certificatePinnerBuilder.add(extractBaseUrl(baseUrl), pubKey)
     }
     return certificatePinnerBuilder.build()
 }
 
-private fun extractBaseUrl(baseUrl: String): String {
+@VisibleForTesting
+internal fun extractBaseUrl(baseUrl: String): String {
     return try {
         val url = URI.create(baseUrl).toURL()
         url.authority
-    } catch (e: MalformedURLException) {
+    } catch (e: IllegalArgumentException) {
```

Comment:
Do you think, we can catch both? 🤔

Reply:
@khairul-alam-licon yes we can, updated it!! thanks 👍